libglusterfs/src/common-utils.c

```diff
@@ -5485,3 +5478,9 @@ get_xattrs_to_heal()
 {
     return xattrs_to_heal;
 }
+
+char *
+gf_gethostname(void)
+{
+    return THIS->ctx->hostname;
```

Comment:
nit: can we call global_ctx instead of THIS? Something in the lines of:

Reply:
Yes.